api.js

```diff
@@ -28,8 +28,14 @@ function Api(options) {
 	EventEmitter.call(this);
 
 	this.options = options || {};
-	this.options.require = (this.options.require || []).map(resolveCwd);
 	this.options.match = this.options.match || [];
+	this.options.require = (this.options.require || []).map(function (module) {
```

Comment:
Minor nit, maybe `moduleId` rather than `module`? `module` is in the file scope as the module object. `resolve-cwd` [uses `moduleId` as well](https://www.npmjs.com/package/resolve-cwd#resolvecwdmoduleid).